Cookbook/Proposals/SplitOctopus.md

```diff
@@ -0,0 +1,45 @@
+# Split Octopus codebase
+
+* Author(s): Giorgos Tsiapaliokas
+* Review Managers: David Rodrigues, Yasuhiro Inami, Olivier Halligon
+
+## Introduction
+
+From the early versions of the codebase Octopus was the place were features were added.
+Unfortunately Octopus hasn't adapted as the rest of the codebase has.
+In the iOS codebase we use frameworks in order to organize the code and Octopus isn't one.
+
+## Motivation
+
+While there is still the need for having a place for small features Octopus has become a massive container which contains both features and services.
+Also from a tech POV Octopus is just a folder which is being reused in different targets.
+I am sure we can do better than this :)
+
+## Proposed solution
+
+Split Octopus into two different frameworks.
+The proposed names are `BabylonFeatures` and `BabylonApp`.
```

Comment:
We already have a `Babylon` target for the app, which `Octopus` folder is part of (as well as all other app targets)  is it the same?